src/app/design-tokens.css

```
@@ -1,23 +1,106 @@
+
+  /* Border Radii */
+  --radius-xs: 4px;
   --radius-sm: 6px;
   --radius-md: 10px;
   --radius-lg: 14px;
+  --radius-xl: 18px;
+  --radius-2xl: 24px;
+  --radius-pill: 9999px;
+
+  /* Typography Scale */
+  --sn-font-family: Inter, ui-sans-serif, system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif;
```

Stylelint (17.14.0)

[error] Expected "Inter" to be "inter" (value-keyword-case)

(value-keyword-case)